api/graphql/schema.graphql

```diff
@@ -142,6 +142,39 @@ type Department {
     name: LocalizedString
 }
 
+enum Status {
+    PENDING
+    DONE
+}
+
+type PoolCandidateFilter {
+    id: ID!
+    classifications: [Classification]
+    cmoAssets: [CmoAsset]
+    hasDiploma: Boolean @rename(attribute: "has_diploma")
+    hasDisability: Boolean @rename(attribute: "has_disability")
+    isIndigenous: Boolean @rename(attribute: "is_indigenous")
+    isVisibleMinority: Boolean @rename(attribute: "is_visible_minority")
+    isWoman: Boolean @rename(attribute: "is_woman")
+    languageAbility: LanguageAbility @rename(attribute: "language_ability")
+    operationalRequirements: [OperationalRequirement] @rename(attribute: "operational_requirements")
+    workRegions: [WorkRegion] @rename(attribute: "work_regions")
+    pools: [Pool]
+}
+
+type PoolCandidateSearchRequest {
+    id: ID!
+    fullName: String! @rename(attribute: "full_name")
+    email: String!
+    department: Department! @hasOne
+    jobTitle: String! @rename(attribute: "job_title")
+    additionalComments: String @rename(attribute: "additional_comments")
+    poolCandidateFilterId: PoolCandidateFilter! @hasOne @rename(attribute: "pool_candidate_filter_id")
+    requestedDate: Date! @rename(attribute: "requested_date")
+    status: Status!
```

Comment:
Unless we have a good reason to, lets leave these fields without the `!` requirement. We can still require all these fields on the inputs for mutations, when creating or updating the model.